src/sub-blocks/Content/Content.tsx

```diff
@@ -65,7 +71,12 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
     const hasTitle = Boolean(title);
 
     return (
-        <Col className={b({size, centered, theme}, className)} reset sizes={colSizes}>
+        <Col
+            className={b({size, centered, theme}, className)}
+            reset
+            sizes={colSizes}
+            qa={qas.container}
```

Comment:
where is container in `qas`? or is it exists by default?

Reply:
'container', 'content', 'wrapper', 'image', 'button' - are in attributes by default. You are right